Libplanet/Blocks/Block.cs

```diff
@@ -40,11 +40,11 @@ public class Block<T> : ISerializable
             Miner = miner;
             PreviousHash = previousHash;
             Timestamp = timestamp;
-            Transactions = transactions.ToArray();
+            Transactions = transactions.OrderBy(tx => tx.Id).ToArray();
```

Comment:
Is this line required?

Reply:
In my thought, we can't trust the transactions are ordered by TxId.
So I added the line :thinking:.

Reply:
Can't we squash this to [line 47](https://github.com/planetarium/libplanet/pull/511/files#diff-e971906424c095136a07b8eefbdb6511R47)?

Reply:
We need to order transactions before calculation of `Hash` because the order of transactions affects to `Hash`. I'm not sure I understood your comment :sweat_smile:

Reply:
Oh, I didn't realize that. Thanks for your comment.